src/Core/src/Platform/iOS/LayoutView.cs

```diff
@@ -68,9 +74,6 @@ public override void WillRemoveSubview(UIView uiview)
 			Superview?.SetNeedsLayout();
 		}
 
-		internal Func<double, double, Size>? CrossPlatformMeasure { get; set; }
-		internal Func<Rect, Size>? CrossPlatformArrange { get; set; }
```

Comment:
I removed these, because you can just call `View.CrossPlatformMeasure()` directly.

Otherwise we might need to unset them in places, like in 7d0af63c8a31f2924bd124a9beccdeadb4648b6a.

Reply:
> I removed these, because you can just call `View.CrossPlatformMeasure()` directly.

The design intent here was for the handlers to be able to directly modify the xplat measure/arrange methods directly when necessary. It's not currently used on iOS (mostly for accidental historical reasons), but the other platforms _do_ make use of it. The intent was to keep the designs symmetric on all the platforms. 

(The main place you can see this at work in the Android and Windows ScrollViewHandlers.)

So I'd rather not remove these. Or, if there's a compelling memory-leak reason to remove them, we'll need to work out a way to inject custom xplat measure/arrange methods from the handlers.

Reply:
> The design intent here was for the handlers to be able to directly modify the xplat measure/arrange methods directly when necessary.

Is this design documented? So other people will know about it?

> Or, if there's a compelling memory-leak reason to remove them, we'll need to work out a way to inject custom xplat measure/arrange methods from the handlers.

The leak appears to return if I put them back. It makes the shell & navigation `DoNotLeak` tests fail. Will share a `*.gcdump` file tomorrow.

Reply:
> Is this design documented? So other people will know about it?

Clearly not well enough, if at all. 🙂